app/controllers/scc_accounts_controller.rb

```diff
@@ -19,6 +19,7 @@ def index
   def new
     @scc_account = SccAccount.new
     @scc_account.organization = @organization
+    find_available_gpg_keys
```

Comment:
use a before_action please instead of calling the function directly. 
See

       before_action :find_resource, only: %i[show edit update destroy sync bulk_subscribe]

Reply:
This does not work for the `new` method because the @scc_account is not set. But I could try to rewrite the `find_available_gpg_keys` in a way that it looks for the @scc_account.organization and uses @organization if @scc_account is not set.

Reply:
Fixed it in #57